libtiledbvcf/src/stats/variant_stats_reader.h

```diff
@@ -51,41 +52,59 @@ class AFMap {
    * @param allele Allele value
    * @param ac Allele Count
    */
-  void insert(uint32_t pos, const std::string& allele, int ac) {
+  void insert(
+      uint32_t pos,
+      const std::string& allele,
+      int ac,
+      int an = 0,
+      uint32_t end = 0) {
     // Should this insertion be tallied for contributing to transport (Arrow)
```

Comment:
Does this comment need to move or be removed?

Reply:
It describes the variable `should_tally`